### ISSUE TITLE
mAP drops suddenly while loss is unchanged

### ISSUE THREAD

**Issue by hoangphucITJP** [AlexeyAB/darknet#5160]
Hi. As stated, the mAP suddenly drops to zero while the loss is unchanged. What could be the problem here?

![image](https://user-images.githubusercontent.com/14938527/78317460-8c6c4b00-758c-11ea-856b-664c493b4345.png)

**Comment by AlexeyAB** [comment]
* What cfg-file do you use?
* Looks like an overfitting, or your train-dataset doesn't correspond to your valid-dataset

https://github.com/AlexeyAB/darknet#how-to-improve-object-detection

> for each object which you want to detect - there must be at least 1 similar object in the Training dataset with about the same: shape, side of object, relative size, angle of rotation, tilt, illumination. So desirable that your training dataset include images with objects at diffrent: scales, rotations, lightings, from different sides, on different backgrounds - you should preferably have 2000 different images for each class or more, and you should train 2000*classes iterations or more
